docs/configs/bridges-2.yaml

```diff
@@ -0,0 +1,32 @@
+engine:
+    max_workers_per_node: 2
+    worker_debug: False
+
+    address:
+      type: address_by_interface
+      ifname: ens3f0
+
+    provider:
+        type: SlurmProvider
+        partition: RM
+
+        launcher:
+            type: SrunLauncher
+
+        # string to prepend to #SBATCH blocks in the submit
+        # script to the scheduler
+        # e.g., "#SBATCH --constraint=knl,quad,cache"
```

Comment:
I'd recommend changing this string to a more GPU specific example string like:
```#SBATCH --gpus=v100-32:8``` from the docs (https://www.psc.edu/resources/bridges-2/user-guide-2-2/#gpu-job)